frontend/talentsearch/src/js/types/PoolUtils.ts

```diff
@@ -0,0 +1,8 @@
+import { Classification } from "@common/api/generated";
```

Comment:
I think our convention is that ts files that aren't components should have camelCase file names, ie `poolUtils.ts`.